packaging/container/Dockerfile

```diff
@@ -1,13 +1,61 @@
+FROM gitlab-registry.cern.ch/cernvm/build-images/centos_x86_64:8 as build-base
+
+FROM build-base as builder
+ARG CVMFS_COMMIT=cvmfs-2.7.4
```

Comment:
This should be unset so that we enforce a provided argument.  Otherwise we might get "silently" the wrong version built.